diff --git a/src/backend/winit.rs b/src/backend/winit.rs
index beace1dc..2619c11f 100644
--- a/src/backend/winit.rs
+++ b/src/backend/winit.rs
@@ -41,6 +41,7 @@ impl Winit {
 
         let builder = Window::default_attributes()
             .with_inner_size(LogicalSize::new(1280.0, 800.0))
+            .with_decorations(false)
             // .with_resizable(false)
             .with_title("niri")
             .with_name("niri", "");
